api/users.go

```diff
@@ -725,15 +726,33 @@ func (r usersRoutes) updateSeekingTime(c *gin.Context) {
 }
 
 func (r usersRoutes) updateAutoSkip(c *gin.Context) {
-	u := c.MustGet("TUMLiveContext").(tools.TUMLiveContext).User
-	if u == nil {
+	u := getUserFromContext(c)
+	var req struct{ Value model.AutoSkipSetting }
+	if err := c.BindJSON(&req); err != nil {
```

Comment:
I think we should still check if the user is null because else the method should already be returned

Reply:
Yeah makes sense. This is the case for a lot of functions in this file so they should probably also be changed.